analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ConfiguringLoggersTest.cs

```diff
@@ -33,60 +33,75 @@ namespace SonarAnalyzer.UnitTest.Rules
+                .Verify();
 
         [TestMethod]
         public void ConfiguringLoggers_NLog_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\Hotspots\ConfiguringLoggers_NLog.cs",
-                new CS.ConfiguringLoggers(AnalyzerConfiguration.AlwaysEnabled),
-                NLogReferences);
+            builderCS.AddPaths(@"Hotspots\ConfiguringLoggers_NLog.cs")
+                .AddReferences(NLogReferences)
+                .Verify();
 
         [TestMethod]
         public void ConfiguringLoggers_NLog_VB() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\Hotspots\ConfiguringLoggers_NLog.vb",
-                new VB.ConfiguringLoggers(AnalyzerConfiguration.AlwaysEnabled),
-                NLogReferences);
+            builderVB.AddPaths(@"Hotspots\ConfiguringLoggers_NLog.vb")
+                .AddReferences(NLogReferences)
+                .Verify();
 
         [TestMethod]
         public void ConfiguringLoggers_Serilog_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\Hotspots\ConfiguringLoggers_Serilog.cs",
-                new CS.ConfiguringLoggers(AnalyzerConfiguration.AlwaysEnabled),
-                SeriLogReferences);
+            builderCS.AddPaths(@"Hotspots\ConfiguringLoggers_Serilog.cs")
+                .AddReferences(SeriLogReferences)
+                .Verify();
 
         [TestMethod]
         public void ConfiguringLoggers_Serilog_VB() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\Hotspots\ConfiguringLoggers_Serilog.vb",
-                new VB.ConfiguringLoggers(AnalyzerConfiguration.AlwaysEnabled),
-                SeriLogReferences);
+            builderVB.AddPaths(@"Hotspots\ConfiguringLoggers_Serilog.vb")
+                .AddReferences(SeriLogReferences)
+                .Verify();
 
 #if NET
         [TestMethod]
-        public void ConfiguringLoggers_AspNetCore_CS() =>
-            OldVerifier.VerifyNonConcurrentAnalyzer(@"TestCases\Hotspots\ConfiguringLoggers_AspNetCore.cs",
-                                    new CS.ConfiguringLoggers(AnalyzerConfiguration.AlwaysEnabled),
-                                    AspNetCoreLoggingReferences);
+        public void ConfiguringLoggers_AspNetCore2_CS() =>
+            builderCS.AddPaths(@"Hotspots\ConfiguringLoggers_AspNetCore.cs")
+                .AddReferences(AspNetCore2LoggingReferences)
+                .WithConcurrentAnalysis(false)
+                .Verify();
+
+        [TestMethod]
+        public void ConfiguringLoggers_AspNetCore6_CS() =>
+            builderCS.AddPaths(@"Hotspots\ConfiguringLoggers_AspNetCore6.cs")
+                .AddReferences(AspNetCoreLoggingReferences(Constants.DotNetCore600Version))
+                .Verify();
+
+        [TestMethod]
+        public void ConfiguringLoggers_AspNetCoreLatest_CS() =>
+            builderCS.AddPaths(@"Hotspots\ConfiguringLoggers_AspNetCore6.cs")
+                .AddReferences(AspNetCoreLoggingReferences(Constants.NuGetLatestVersion))
+                .Verify();
```

Comment:
Is there a difference between ver 6 and the Latest? I think we should have a single test here. For ConfiguringLoggers_AspNetCoreLatest_CS

Reply:
My understanding was that we'd have one test with a stable version (in this case I build the example on v6) and then one more test getting the latest version to see if we have breaking changes with newer versions.

Otherwise, you propose to have a test only for the latest version?

Reply:
The idea is to have tests only where the behavior changes: e.g. if the APIs would have changed in .Net 5 we would have added one test for Asp.Net Core 5 and one with Latest (Asp .Net Core 6).

In this case, both tests, .Net 6 and latest check the same result so we can have only the Latest test.